src/tools/ScenarioMeasurement/ScenarioMeasurement.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Do you want to include the `..\..\Reporting\Reporting\Reporting.csproj` to the solution (included in `Startup.csproj`)?

Reply:
Good question. I went back and forth on this, as logically `Reporting` is an independent component used elsewhere in the system. That said, the weirdness of how the project system works differently in VS versus the CLI is annoying. I'll put it in there.

Reply:
The way the `csproj` project dependencies are treated and not added automatically in the solution is weird.